### DISCUSSION TITLE
How do I match a specific ASCII code point?

### DISCUSSION THREAD

**Discussion by dertuxmalwieder** [pest-parser/pest#919] [Q&A] [answered]
I know that character ranges are [a solved problem](https://github.com/pest-parser/pest/discussions/657), but the parser does not accept single characters: "expected '..'". How can I make a rule that only matches the LF character? I tried this:

```
linefeed = { '\u{0A}' }
```

Sadly, according to your own testbed on pest.rs, this isn't a valid rule. What would be?

**Comment by tomtau** [comment, marked as answer]
```
linefeed = { "\n" }
```

or

```
linefeed = { '\u{0A}'..'\u{0A}' }
```

?

**Reply by dertuxmalwieder** [reply]
Thank you, the "one character range" works well. I still wonder whether it makes sense *at all* to enforce that.
